io.catenax.shared.recycling_strategy_certificate/2.0.0/metadata.json

```diff
@@ -0,0 +1 @@
+{ "status" : "standardized"} 
```

Comment:
If this is new model, the the status should be 'release'. i.e. `{ "status" : "release"} `

Reply:
copy that.